docs/extensions/intro.md

```diff
@@ -0,0 +1,170 @@
+# Introduction
+
+Extensions is a way to extend Specter Desktop in a way which does not need PRs in the core-project. They can be used
+to 
+* create elaborated Frontends on top of Specter Desktop or even replace the whole Frontend
+* Add Additional Menu Items currently in Wallets and Settings
+* Create additional Devices with the functionality you would like to have
```

Comment:
The Markdown for a list item is a hyphen, not an asterisk.

Reply:
You're making me angry! Anyway, after calming down i finally understood some of the strange effects which puzzled my markdown life and also what wishful thinking can do to the world!
Just kidding, thanks, very good point ;-)